application/controller/RegisterController.php

```diff
@@ -46,11 +46,12 @@ public function register_action()
 
     /**
      * Verify user after activation mail link opened
-     * @param int $user_id user's id
      * @param string $user_activation_verification_code user's verification token
      */
-    public function verify($user_id, $user_activation_verification_code)
+    public function verify($user_activation_verification_code)
     {
+		$user_id = Encryption::decrypt(Request::get("user_id"));
```

Comment:
Request::get("user_id") should be check for empty value otherwise exception is thrown